go/vt/vtgate/planbuilder/builder.go

```diff
@@ -35,6 +36,10 @@ import (
 	vtrpcpb "vitess.io/vitess/go/vt/proto/vtrpc"
 )
 
+var (
+	enableOnlineDDL = flag.Bool("enable_online_ddl", true, "Allow users to submit, review and control Online DDL")
```

Comment:
I don't think we want to have flags all the way in here. Most vtgate flags live [here](https://github.com/vitessio/vitess/blob/master/go/vt/vtgate/vtgate.go#L56).

Reply:
@systay right! I began that way, but then got into a circular `import` error, since `vtgate` package imports `planbuilder` package, thus I can't reference `vtgate` package back from `planbuilder`. Any ideas are welcome.